app/src/main/kotlin/com/exallium/todoapp/allnotes/AllNotesPresenter.kt

```diff
@@ -25,4 +32,10 @@ class AllNotesPresenter(private val view: AllNotesView,
         adapter.cleanup()
     }
 
-}
\ No newline at end of file
+    private fun makeNoteDetailBundle(id: String?) : Bundle {
+        val args: Bundle = bundleFactory.createBundle()
+        screenBundleHelper.setNoteId(args, id)
+        screenBundleHelper.setTitle(args, R.string.note_detail_screen_title)
```

Comment:
I think that from a purity point of view, it might be better to expose that string via the view.  Think of it as your interface into the resources.

But since it's static, I don't really care.